keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/AdminControllerTests.kt

```diff
@@ -0,0 +1,159 @@
+package com.netflix.spinnaker.keel.rest
+
+import com.netflix.spinnaker.keel.KeelApplication
+import com.netflix.spinnaker.keel.api.DeliveryConfig
+import com.netflix.spinnaker.keel.pause.ActuationPauser
+import com.netflix.spinnaker.keel.persistence.KeelRepository
+import com.netflix.spinnaker.keel.persistence.memory.InMemoryDeliveryConfigRepository
+import com.netflix.spinnaker.keel.spring.test.MockEurekaConfiguration
+import com.netflix.spinnaker.time.MutableClock
+import com.ninjasquad.springmockk.MockkBean
+import dev.minutest.junit.JUnit5Minutests
+import dev.minutest.rootContext
+import io.mockk.clearAllMocks
+import java.time.Clock
+import java.time.Instant
+import java.time.ZoneId
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.MediaType
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers
+
+@ExtendWith(SpringExtension::class)
+@SpringBootTest(
+  classes = [KeelApplication::class, MockEurekaConfiguration::class],
+  webEnvironment = SpringBootTest.WebEnvironment.MOCK
+)
+@AutoConfigureMockMvc
+internal class AdminControllerTests : JUnit5Minutests {
+  @Configuration
+  class TestConfiguration {
+    @Bean
+    fun clock(): Clock = MutableClock(
+      Instant.parse("2020-03-25T00:00:00.00Z"),
+      ZoneId.of("UTC")
+    )
+  }
+
+  @MockkBean
+  lateinit var authorizationSupport: AuthorizationSupport
```

Comment:
None of the `AdminController` APIs are authorized (which is a bit scary) so you don't need this...

Reply:
should it be authorized?